react-common/components/controls/Feedback/Feedback.tsx

```diff
@@ -6,48 +6,55 @@ import { Modal } from "../Modal";
 
 // both components require onClose because the feedback modal should close when the user clicks the "finish" button
 // this would not happen if the EventListener did not have a callback to close the modal
-interface IFeedbackModalProps {
-  feedbackConfig: ocv.IFeedbackConfig;
+interface IFeedbackProps  {
+  kind: ocv.FeedbackKind;
   frameId: string;
-  title: string;
-  onClose: () => void;
+  onClose?: () => void;
 }
 
 // right now, there are two kinds of feedback that I think could be valuable for our targets
 // generic and rating feedback, but we will likely want to expand this
-interface IFeedbackProps {
+interface IFeedbackModalProps {
   kind: ocv.FeedbackKind;
   onClose: () => void;
 }
 
 // Wrapper component of the feedback modal so kind can determine what feedback actually shows in the modal
-export const Feedback = (props: IFeedbackProps) => {
+export const FeedbackModal = (props: IFeedbackModalProps) => {
   const { kind, onClose } = props;
+  const title = kind === "generic" ? lf("Leave Feedback") : lf("Rate this activity");
+
   return (
-    <>
-      {kind === "generic" &&
-        <FeedbackModal
-        feedbackConfig={baseConfig}
-        frameId="menu-feedback-frame"
-        title={lf("Leave Feedback")}
-        onClose={onClose}
-      />}
-      {kind === "rating" &&
-        <FeedbackModal
-        feedbackConfig={ratingFeedbackConfig}
-        frameId="activity-feedback-frame"
-        title={lf("Rate this activity")}
-        onClose={onClose} />
-      }
-    </>
+    <Modal className="feedback-modal" title={title} onClose={onClose}>
+      <>
+        { kind === "generic" &&
+          <Feedback
```

Comment:
If the differences between these two feedback components are fully based on the `kind` field, and if that `kind` field has to be passed into the Feedback component anyway, what's the motivation to split them here rather than having the `kind`-based checks inside the Feedback component itself? (In other words, could we set `frameId` inside Feedback based on kind? Then I don't think we'd need this split creating different Feedback components based on it).

Another approach would just be to have:
```
const frameId = kind === "rating" ? "activity-feedback-frame" : "menu-feedback-frame";
<Feedback kind={kind} frameId={frameId} onClose={onClose} />
```